Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -82,6 +82,46 @@ An impact defines the relation between an incident and a component. A component
 
 To reflect this, each component and incident can have a list of impacts, stating the type of impact and a reference to the incident or component, it refers to.
 
+Furthermore a severity value is added to the impact when referencing an component, to gauge the impacts severity on the system.
+
+### Severity
+
+A severity contains a name, that MUST be unique and WILL be used as identifier. The value marks the upper boundary of the severity.
+
+The severity value range is calculated by the previous severitys (SeverityA) value + 1 to the current severitys (SeverityB) value. These limits are inclusive.
```

Comment:
Typo. Fixed:

``The severity value range is calculated by taking the previous severity's (`SeverityA`) value and adding 1 to obtain the starting point and taking the current severity's (SeverityB) value as the end point. These limits are inclusive.``

Reply:
The `Phase`s are referenced by their name too, in combination with their generation, to uniquely identify them. There is no UUID used either. 

Giving the severities an UUID is only required, if two or more severities with the same name should be possible.

Forcing uniqueness in the names, seems a lot more logical, then having severities with the same name. When names are unique, they can be used as identifier.

A possible future requirements is i18n, but the whole API isn't set up to handle different languages, e.g. Phases. Currently only one language at a time is supported.

I18n would probably be out of scope for an MVP anyways, but maybe we should check back with @fkr about this?

Reply:
Sounds good, we'll discuss this with him during the sprint review later today.

Aside from localization, the use-case I could imagine for non-unique severity names would be setting up several severities of the same name for public consumption, while the different severity ranges could be used for internal purposes, such as reporting. I am, however, uncertain how realistic this scenario is.

Reply:
As discussed in the sprint review the i18n feature is out of scope for the MVP.

Severities will use their unique name as identifier.